src/goto-programs/goto_function.h

```diff
@@ -115,9 +115,10 @@ class goto_functiont
   ///
   /// The validation mode indicates whether well-formedness check failures are
   /// reported via DATA_INVARIANT violations or exceptions.
-  void validate(const namespacet &ns, const validation_modet vm) const
+  void validate(const symbol_tablet &table, const validation_modet vm) const
```

Comment:
We check that the declared/removed symbols are known to the symbol table. It might also be a part of #3118. Eventually we will need `goto_programt::instructiont::validate` to have access to `symbol_table` somehow.

Reply:
```namespacet``` is the usual wrapper for a ```symbol_table``` and is (or at least was, last time I checked) the preferred interface.

Reply:
Good to know, I'll revert the changes to pass `namespacet` only.

Reply:
Ta.